src/encoded/ingestion/loadxl_extensions.py

```diff
@@ -0,0 +1,79 @@
+            # POST: beefcafe-01ce-4e61-be5d-cd04401dff29
+            # PATCH: deadbabe-7b4f-4923-824b-d0864a689bb
+            # SKIP: feedbeef-eb17-4406-adb8-060ea2ae2180
+            # CHECK: cafebabe-eb17-4406-adb8-0eacafebabe
+            # ERROR: deadbeef-483e-4a08-96b9-3ce85ce8bf8c
+            # Note that SKIP means skip POST (create); it still may do PATCH (update), if overwrite.
+            if isinstance(item, bytes):
+                item = item.decode("ascii")
+            elif not isinstance(item, str):
+                continue
+            match = LOADXL_RESPONSE_PATTERN.match(item)
+            if not match or match.re.groups != 2:
+                continue
+            action = LOADXL_ACTION_NAME[match.group(1).upper()]
+            identifying_value = match.group(2)
+            if not response.get(action):
+                response[action] = []
+            response[action].append(identifying_value)
+            unique_identifying_values.add(identifying_value)
+        # Items flagged as SKIP in loadxl could ultimately be a PATCH (update),
+        # so remove from the skip list any items which are also in the update list. 
+        response["skip"] = [item for item in response["skip"] if item not in response["update"]]
+        # Items flagged as POST (create) in loadxl typically also are flagged as PATCH (update), due to the
+        # way they are written, so remove from the update list any items which are also in the create list.
+        response["update"] = [item for item in response["update"] if item not in response["create"]]
+        response["unique"] = len(unique_identifying_values)
+        response["ntypes"] = len(data)
+        return response
+
+    loadxl_load_data_response = loadxl_load_data(
+        testapp=portal_vapp,
+        inserts=data,
+        docsdir=None,
+        overwrite=True,
+        itype=None,
+        from_json=True,
+        patch_only=False,
+        validate_only=validate_only)
+
+    return package_loadxl_response(loadxl_load_data_response)
+
+
+def summarize_load_data_into_database_response(load_data_response: Optional[dict],
+                                               submission: SmahtSubmissionFolio) -> list[str]:
+    """
+    Summarize the given load data results into a simple short list of English phrases;
+    this will end up going into the additional_properties of the IngestionSubmission
+    object in the Portal database (see SubmissionFolio.record_results); this is what will
+    get displayed, by default, by the submitr tool when it detects processing has completed.
+    """
+    return [
+        f"Successful ingestion summary:",
+        f"In File: {submission.data_file_name}",
+        f"S3 File: {submission.s3_data_file_location}",
+        f"Details: {submission.s3_details_location}",
+        f"N Types: {load_data_response['ntypes']}",
+        f"Uniques: {load_data_response['unique']}",
```

Comment:
Like mentioned today, this may be very long, you might want to just give a number?

Reply:
Indeed. Having struggled with maintaining this in detail, I think we might want to consider testing merely that certain key lines are present, so having a list of regexps that are expected to appear at all, or in a given order. Probably if some of them are there, the others are.